fcrepo-export/src/main/java/org/fcrepo/exporter/Exporter.java

```diff
@@ -84,13 +85,13 @@ private void export(final URI uri) {
             if (linkHeaders.contains(binaryURI)) {
                 exportBinary(uri);
             } else if (linkHeaders.contains(containerURI)) {
-                exportDescription(uri);
+                exportDescription(uri, uri);
```

Comment:
I think I get what's happening here, but this is so odd looking that it might be worth adding
```
/**
 * Some kind of explanation as to why this makes sense.
 */
private void exportDescription(final URI uri) { exportDescription(uri, uri); } 
```